src/components/Auth/UserFormDialog/AddUserDialog.test.tsx

```diff
@@ -31,30 +27,39 @@ describe('AddUserDialog', () => {
     return {
       triggerValidation,
       onClose,
-      addUser,
+      createUser,
       ...methods,
     };
   }
 
   it('calls onClose on form submit', async () => {
-    const { triggerValidation, onClose } = await setup();
+    const { triggerValidation, onClose, getByLabelText } = await setup();
     await waitForDialogsToOpen();
+
+    const input = getByLabelText(/Phone authentication/) as HTMLInputElement;
+
+    fireEvent.change(input, {
+      target: { value: '+1 689-689-6896' },
```

Comment:
nit: please change to a 555-number.

Reply:
What's 555 number?

Reply:
Sorry, to clarify, I meant to switch to a [fictional telephone number](https://en.wikipedia.org/wiki/555_(telephone_number)), usually one with 555 as the code. e.g. `+15555550100`. In this way, we can ensure that nobody gets spammed.

> Only 555-0100 through 555-0199 are now specifically reserved for fictional use